src/ServiceStack/DependencyInjection/DependencyService.cs

```diff
@@ -75,25 +137,7 @@ public void UpdateRegistrations()
                 _containerBuilder = new ContainerBuilder();
             }
         }
-
-        private void SetSharing(IRegistrationBuilder<object, object, SingleRegistrationStyle> registration,
-                                Sharing sharing)
-        {
-            switch (sharing)
-            {
-                case Sharing.None:
-                    registration.InstancePerDependency();
-                    break;
-                case Sharing.PerRequest:
-                    registration.InstancePerMatchingLifetimeScope("httpRequest");
-                    break;
-                case Sharing.Singleton:
-                    registration.RegistrationData.Sharing = InstanceSharing.Shared;
-                    registration.RegistrationData.Lifetime = new RootScopeLifetime();
```

Comment:
Is there a behavioral difference between this singleton registration and the one above?